cli.py

```diff
@@ -68,18 +71,35 @@ def main(log_level):
               type=str, help="D3A domain name")
 @click.option('-w', '--web-socket', default=DEFAULT_WEBSOCKET_DOMAIN,
               type=str, help="D3A websocket URL")
-@click.option('--run-on-redis', is_flag=True, default=False, help="Start the client using the Redis API")
-def run(setup_module_name, username, password, domain_name, web_socket, run_on_redis, **kwargs):
+@click.option('-i', '--simulation-info', type=str,
+              help="Simulation File info (accept absolute and relative path)")
+@click.option('--run-on-redis', is_flag=True, default=False,
+              help="Start the client using the Redis API")
+def run(base_setup_path, setup_module_name, username, password, domain_name, web_socket,
+        simulation_info, run_on_redis, **kwargs):
     if username is not None:
         os.environ["API_CLIENT_USERNAME"] = username
     if password is not None:
         os.environ["API_CLIENT_PASSWORD"] = password
     os.environ["API_CLIENT_DOMAIN_NAME"] = domain_name
     os.environ["API_CLIENT_WEBSOCKET_DOMAIN_NAME"] = web_socket
     os.environ["API_CLIENT_RUN_ON_REDIS"] = "true" if run_on_redis else "false"
+    if os.path.isabs(simulation_info):
```

Comment:
A very small note on this: if you want, you could use the [`pathlib`](https://docs.python.org/3.8/library/pathlib.html) module to replace `os.path`. I think it is a bit cleaner and I have the feeling that's the direction in which the python community has been going for a while. (For example: [Why you should be using pathlib](https://treyhunner.com/2018/12/why-you-should-be-using-pathlib)).
Not a big deal anyway.

Reply:
@spyrostz @hannesdiedrich Would you guys prefers pathlib. Im indifferent actually